portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -57,11 +58,23 @@
   private static final String NEWEST_FIRST = "Newest First";
   private static final String LONGEST_FIRST = "Longest First";
 
+  private static final String ID = "ID";
+  private static final String USER_INFO = "userInfo";
+
+  private static DatastoreService datastore;
+  
+  @Override
+  public void init() {
+    this.datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int maxComments = Integer.parseInt(request.getParameter(COMMENT_AMOUNT));
     String sort = request.getParameter(SORT);
 
+    // Move addSort() code to its own method.
```

Comment:
Should this be marked as a TODO?

Reply:
Yes, thank you for catching that!

Done!